### ISSUE TITLE
How to input values for added custom id?

### ISSUE THREAD

**Issue by Lava1992** [rdbrck/jira-template-injector#58]
Hi, 
I can successfully create custom ID but I don't know how to input value in the e.g. environment field via the template?
Applying template will only work for the pre defined "Description" field.

Would be nice if someone can give me an example on how to put in values also for the added customId's.

Thank you in advance,
Lava1992

**Comment by asundberg-rdbrck** [comment]
You are correct that the template will currently only work on the Description field.

This is something we will consider for future releases.

Thanks for the feedback.

**Comment by icakirlar** [comment]
I would vote it as a top 1 new feature as a user.

**Comment by shako** [comment]
This feature request gets my vote too!

**Comment by dquimper** [comment]
What can we do with the "custom input ID" once we added them? 
Can someone add an explanation for this feature?

**Comment by shako** [comment]
Something like: it lets the user prefill a custom field.

**Comment by dquimper** [comment]
I mean what can we do with the feature today? I get that it could be used to prefill custom fields in the future.
